src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -79,14 +79,13 @@ object GermlineAssemblyCaller {
      * @param reads  Set of reads to extract sequence from
      * @param startLocus Start (inclusive) locus on the reference
      * @param endLocus End (exclusive) locus on the reference
-     * @param kmerSize Length of the subsequence
      * @param minOccurrence Mininum number of times a subsequence needs to appear to be included
      * @return List of subsequences overlapping [startLocus, endLocus) that appear at least `minOccurrence` time
      */
     def getConsensusKmer(reads: Seq[MappedRead],
                          startLocus: Int,
                          endLocus: Int,
-                         minOccurrence: Int): Iterable[Array[Byte]] = {
+                         minOccurrence: Int): Iterable[ContigSequence] = {
```

Comment:
Now that `ContigSequence` is an alias for `IndexedSeq`, I am more in favor of having it (or an alias like it) here instead of `Array[Byte]`, as a way of enforcing that genomic sequences that we index in to are efficiently indexable.

Do you want me to rename it to just `Sequence`? `GenomicSequence`? Something else? Leave it as `ContigSequence`?